packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -120,33 +142,49 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
         gltf,
         gltfEmissiveTexture ? gltfEmissiveTexture : null,
     );
+
+    // Releases the gltf info.
+    this[$lazyLoadGLTFInfo] = undefined;
+  }
+
+  private[$ensureLoaded]() {
+    if (this[$lazyLoadGLTFInfo] !== undefined) {
+      this[$initialize](this[$lazyLoadGLTFInfo]!);
+    }
   }
 
   get name(): string {
+    this[$ensureLoaded]();
```

Comment:
Calling `ensureLoaded` makes sure the material is properly configured. However, not strictly necessary, any operation done on the material would succeed, it just might not work as expected, this could be left out and we could assume that a valid material would always have been loaded correctly without having to 'ensure' things everywhere.